tests/test_parameters.py

```diff
@@ -324,6 +325,72 @@ def test_daily_profile_leap_day(model):
     model.run()
     assert_allclose(inpt.flow, 365)
 
+def test_scenario_daily_profile(simple_linear_model):
```

Comment:
Perhaps organise the two `ScenarioDailyProfileParameter` tests into a class for readability.